include/deal.II/matrix_free/mapping_info.templates.h

```diff
@@ -2477,6 +2491,13 @@ namespace internal
                               vv,
                               my_data.jacobians[1][offset + q][d][e]);
                         }
+
+                      if (update_flags_faces & update_jacobian_grads)
+                        {
+                          // TODO: implement jacobian grads for general path
+                          AssertThrow(false, ExcNotImplemented());
```

Comment:
It is not computing these values since `update_flags_faces & update_jacobian_grads` is always false on faces at the moment. I will implement the general path for faces in a follow up PR.